src/components/__tests__/EmailInviteDialog.test.js

```diff
@@ -10,13 +10,57 @@ import SocialShare from 'src/components/SocialShare'
 import { ThemeProvider } from '@material-ui/core/styles'
 import theme from 'src/utils/theme'
 import IconButton from '@material-ui/core/IconButton'
+import shareCats from 'src/assets/images/shareCats.png'
+import catsSent from 'src/assets/images/catsSent.png'
+import seasEmailInvite from 'src/assets/images/seasEmailInvite.svg'
 
 const getMockProps = () => ({
   username: 'someUsername',
   userId: 'someId',
   landingPagePath: '/cats/',
   closeFunction: jest.fn(),
+  user: {
+    cause: {
+      sharing: {
+        imgCategory: 'cats',
+        title: 'Share Tab for Cats with your friends',
+        subtitle:
+          "Save more cats! When a friend signs up, you'll each earn 5 additional treats to help a shelter cat get adopted. 😺",
```

Comment:
I think we want to make all strings that go into typography markdown stored in individual files